urwid/event_loop/asyncio_loop.py

```diff
@@ -67,20 +66,37 @@ class AsyncioEventLoop(EventLoop):
     def __init__(self, *, loop: asyncio.AbstractEventLoop | None = None, **kwargs) -> None:
         super().__init__()
         self.logger = logging.getLogger(__name__).getChild(self.__class__.__name__)
-        if loop:
-            self._loop: asyncio.AbstractEventLoop = loop
+
+        if sys.version_info[:2] < (3, 11):
             self._event_loop_policy_altered: bool = False
             self._original_event_loop_policy: asyncio.AbstractEventLoopPolicy | None = None
```

Comment:
`AbstractEventLoopPolicy` and all logic around event loop policy has been removed in python 3.14